Makefile.am

```diff
@@ -74,7 +74,6 @@ AM_CFLAGS += \
 	-Wall \
 	-Wuninitialized \
 	-Wunused-but-set-parameter \
-	-Wcast-align \
```

Comment:
Do we want to drop this entirely or plan to fix this later? In the latter case, you might open a jira issue for it.